Makefile

```diff
@@ -204,6 +204,8 @@ PUB_C:=$(PUB)/c
 PMC_DIR:=ptp-tools
 HMAC_SRC:=hmac
 CLKMGR_DIR:=clkmgr
+CLKMGR_PUB:=$(CLKMGR_DIR)/pub
+CLKMGR_PUB_C:=$(CLKMGR_PUB)/clkmgr
```

Comment:
What is _C stands for ?

Reply:
clkmgr/pub dint got specific folder for c header, so should not use the name _C